examples/cassandra/cassandra-controller.yaml

```diff
@@ -1,13 +1,17 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  labels:
-    app: cassandra
   name: cassandra
+  # The labels can be applied automatically
+  # from the labels in the pod template, if not set
+  # labels:
+    # app: cassandra
 spec:
   replicas: 2
-  selector:
-      app: cassandra
+  # The selector can be applied automatically
+  # from the labels in the pod template, if not set.
+  # selector:
+      # app: cassandra
   template:
     metadata:
       labels:
```

Comment:
This is just a 'clean' volume (http://kubernetes.io/docs/user-guide/volumes/#emptydir).
As per our email thread, I'm not sure why Brendan specifically did it this way, but we should investigate whether syncing problems are introduced if a cassandra node (a pod) is using a PD, goes down, comes back up, and needs to resync.